dashboard/config/libraries/variableLog.interpreted.js

```diff
@@ -69,3 +69,13 @@ function storeVariableLogforPrevious() {
   previousVarLog = JSON.parse(JSON.stringify(varLog));
 }
 
+// Returns true if the student has a variable label that starts with "_"
+function noBadVariableLabels() {
```

Comment:
nit: I wonder if we can name this more a bit more specific to what this function is doing. Maybe something like `allowBeginningWithUnderscore` or something.

Reply:
Reversed the logic and renamed to `varLabelStartsWithUnderscore()`. Updated each level that used the function.